.github/workflows/build-and-test.yml

```diff
@@ -3243,16 +3320,74 @@ jobs:
     needs: [ build-python-LinuxArmv7, build-python-LinuxAarch64, train-test-model-Linux, build-test-chroot ]
     if: ${{ github.event_name == 'pull_request' }}
     strategy:
+      fail-fast: false
       matrix:
-        arch: [ "armv7", "aarch64" ]
-        python-version: [3.7]
-        models: ["test", "prod"]
-        samplerate: ["8000", "16000"]
+        include:
```

Comment:
same, why changing the way the matrix is defined?

Reply:
Fixed

Reply:
@lissyx Please see the above comment, the same reason.